hatch_jupyter_builder/utils.py

```diff
@@ -94,25 +94,27 @@ def npm_builder(
     if isinstance(npm, str):
         npm = [npm]
 
+    is_yarn = (abs_path / "yarn.lock").exists()
```

Comment:
It's always been possible that this heuristic is incorrect, but the risk is slightly magnified now that we rely on it even if `npm` (the executable location) is explicitly provided.

I don't see a better alternative though, without attempting to look within `npm` to do pattern-matching for `"yarn"`, which could still be incorrect (e.g. `npm = "/home/john.yarn/bin/npm"`).

Reply:
Specifically this change is what I am uncertain about.